using-mobile-extensions/adobe-media-analytics/media-api-reference.md

```diff
@@ -771,9 +771,9 @@ Method to track media events.
 
 | Variable Name | Description                                                                                                                                                                                                                                                                                                                           |
 |---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `event `      | [Media event]#(media-events)                                                                                                                                                                                                                                                                                                          |
-| `info`        | For AdBreakStart, AdBreak information created using [Create AdBreak Object](#create-adbreak-object).<br> For AdStart, Ad information created using [Create Ad Object](#create-ad-object).<br> For ChapterStart, Chapter information created using [Create Chapter Object](#create-chapter-object).<br> Not required for other events. |
-| `data`        | Optional context data only for AdStart and ChapterStart. Not required for other events.                                                                                                                                                                                                                                               |
+| `event `      | [Media event](#media-events)                                                                                                                                                                                                                                                                                                          |
+| `info`        | For AdBreakStart event, AdBreak information created using [Create AdBreak Object](#create-adbreak-object).<br> For AdStart event, Ad information created using [Create Ad Object](#create-ad-object).<br> For ChapterStart event, Chapter information created using [Create Chapter Object](#create-chapter-object).<br> Not required for other events. |
```

Comment:
Please change the sentences so that they read as follows:

- "For the AdBreakStart event, AdBreak information is created using [Create AdBreak Object](#create-adbreak-object)."

- "For the AdStart event, Ad information is created using [Create Ad Object](#create-ad-object).<br> 

- For the ChapterStart event, Chapter information is created using [Create Chapter Object](#create-chapter-object).<br> This is not required for other events.